src/components/overview/sub-Components/imageCarousel/thumbnailImages.jsx

```diff
@@ -13,14 +13,17 @@ const ThumbnailImages = function(props) {
     thumbnailRows.push(row);
   }
 
+  //this is just to give me a random key to stop console warnings
+  let keycounter = 0;
```

Comment:
We spoke about this but a way to clean up this code is to just use a second parameter in map's callback like so... [array].map( (value, idx) => <div key={idx}>value</div>)